en/ros/ros1_via_ros2.md

```diff
@@ -20,14 +20,14 @@ The PX4 development team recommend that all users [upgrade to ROS 2](../ros/ros2
 
 ## Overview
```

Comment:
This all needs to be reviewed and retested to see if it still works. I would even replace this with eProsima's [Integration Service](https://integration-service.docs.eprosima.com/en/latest/) (proposed on one of my talks)